src/schema-association-service.ts

```diff
@@ -93,6 +96,8 @@ async function autoDetectSchema(
     context: vscode.ExtensionContext,
     workspaceFolder: vscode.WorkspaceFolder): Promise<vscode.Uri | undefined> {
     const azureAccountApi = await getAzureAccountExtensionApi();
+    const config = vscode.workspace.getConfiguration('azure-pipelines')
+    var oneesptSchemaEnabled = vscode.workspace.getConfiguration('azure-pipelines').get<string>('1ESPipelineTemplatesSchemaFile', '');
```

Comment:
1. No `var`
2. Why is this a string and not a boolean?

Reply:
Fixed it